lib/tasks/test_publishing_schemas.rake

```diff
@@ -1,7 +1,8 @@
 namespace :test do
   Rake::TestTask.new(:publishing_schemas => "test:prepare") do |t|
     t.libs << 'test'
-    t.test_files = FileList['test/unit/presenters/publishing_api_presenters/*_test.rb']
+    t.test_files = `grep -rl "valid_against_schema" test`.lines.map(&:chomp)
```

Comment:
While checking this I noticed that most of our tests don't run that assertion :(